src/theme/dojo/raised-button.m.css

```diff
@@ -0,0 +1,14 @@
+@import './variables.css';
+
+.root {
+	box-shadow: var(--box-shadow-dimensions-small) var(--color-box-shadow);
```

Comment:
Just a note, this file will need to be modified with the changes of dojo/themes#105 to ensure that the base component styles are included.